functionary/ui/tables/schedule_task.py

```diff
@@ -35,6 +35,7 @@ class ScheduledTaskTable(tables.Table):
         ),
         verbose_name="Scheduled Task",
     )
+    function = tables.Column(accessor="function__name")
```

Comment:
With the accessor change you need to manually set the name, otherwise it will be "Name" rather than "Function".  

```suggestion
    function = tables.Column(accessor="function__name", verbose_name="Function")
```